features/gebruik_in_lopende_tekst.feature

```diff
@@ -1,189 +1,651 @@
+
+  De gebruikInLopendeTekst wordt op dezelfde manier samengesteld als voor een persoon zonder adellijke titel of predikaat in de volgende gevallen:
+  - Wanneer de combinatie van adellijkeTitelPredikaat en geslachtsaanduiding niet voorkomt in bovenstaande tabel: geslachtsaanduiding "O" of een vrouw met de titel "R" (ridder)
+  - Wanneer de persoon de geslachtsnaam van de echtgenoot/partner gebruikt zonder de eigen geslachtsnaam: aanduidingNaamgebruik is "P"
+  - Wanneer de persoon een predikaat heeft en de geslachtsaanduiding is gelijk aan "V" (vrouw) en betrokkene heeft een partner
+  - Wanneer de persoon een predikaat heeft en de geslachtsaanduiding is gelijk aan "V" (vrouw) en betrokkene gebruikt de naam van de (ex)partner (aanduidingNaamgebruik is ongelijk aan "E")
+
+  Abstract Scenario: persoon met adellijke titel heeft aanduidingNaamgebruik "<naamgebruik>"
+    Gegeven het systeem heeft een persoon met de volgende gegevens
+    | naam                | waarde     |
+    | burgerservicenummer | 999992934  |
+    | geslachtsaanduiding | V          |
+    En de persoon heeft de volgende naam gegevens
+    | naam                                 | waarde                    |
+    | voornamen (02.10)                    | Jo Anne                   |
+    | adellijke titel of predikaat (02.20) | BS                        |
+    | voorvoegsel (02.30)                  | van den                   |
+    | geslachtsnaam (02.40)                | Aedel                     |
+    | aanduidingNaamgebruik (61.10)        | <naamgebruik>             |
+    En de partner heeft de volgende naam gegevens
+    | naam                                 | waarde |
+    | adellijke titel of predikaat (02.20) |        |
+    | voorvoegsel (02.30)                  | de     |
+    | geslachtsnaam (02.40)                | Boer   |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 999992934                       |
+    | fields              | naam.gebruikInLopendeTekst      |
+    Dan bevat de persoon met burgerservicenummer '999992934' de volgende 'naam' gegevens
+    | naam                  | waarde                  |
+    | gebruikInLopendeTekst | <gebruikInLopendeTekst> |
+
+    Voorbeelden:
+    | naamgebruik | gebruikInLopendeTekst                 |
+    | E           | barones van den Aedel                 |
+    | P           | mevrouw De Boer                       |
+    | V           | mevrouw De Boer-barones van den Aedel |
+    | N           | barones van den Aedel-de Boer         |
+
+  Abstract Scenario: persoon met predikaat en geslachtsaanduiding "<geslacht>" heeft een partner en aanduidingNaamgebruik "<naamgebruik>"
```

Comment:
ik denk dat deze titel anders moet worden geformuleerd. Toen ik het las dacht ik eerst dat het de aanduiding naamgebruik van de partner betreft

Reply:
Of dezelfde stijl volgen als de scenario hierboven. Daar heeft de persoon ook een partner en dat wordt niet gemeld in de titel

Reply:
snap ik. zo beter?: 'persoon met predikaat en geslachtsaanduiding "<geslacht>" en aanduidingNaamgebruik "<naamgebruik>" heeft een partner'